### PR TITLE
ctrl+clicking firelocks as silicon opens them without popup

### PR DESCRIPTION
**Pull request by robintime**
**imagine the following:**

the survivors of the latest depressurization event are trying to reach escape, but all these pesky firelocks are in their way! you, a good, humble AI player, are trying to open these firelocks, but you get this prompt that's like 'are you SURE???' and it takes you too many seconds to click yes before people just pause and crowbar them open anyways. now you feel like a useless piece of shit because your super special role of interfacing with the station just got replaced by a bent piece of metal. what if i told you it didn't have to be this way?

:cl:
 * rscadd: new hotkey — ctrl+clicking firedoors allows silicons to quickly open them

### CODE REVIEW

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -250,10 +250,10 @@ var/global/list/alert_overlays_global = list()
...
 
 /obj/machinery/door/firedoor/CtrlClick(mob/user)
-	if(isAdminGhost(user))
-		attack_ai(user,TRUE)
-	else
-		..()
+    if(isrobot(user) || isAdminGhost(user))
+        attack_ai(user, TRUE)
+    else
+        ..()
```
Comment (Kurfursten) [inline code/game/machinery/doors/firedoor.dm]: indent with tab instead of space

**Comment by Kurfursten** [comment]
Needs a descriptive changelog

File: code/game/machinery/doors/firedoor.dm (hunk 1 above)
Comment (robintime) [reply code/game/machinery/doors/firedoor.dm]: I DIDN'T EVEN NOTICE THIS WHAT THE HELL? 🤨
fixed this in the latest commit. blame github user mentgah

**Comment by ghost** [comment]
Why cant' it be shift clicking like regular firelocks?

**Comment by robintime** [comment]
> Why cant' it be shift clicking like regular firelocks?

shift clicking firelocks as AI already has a function, which is to give you information on the firelock itself

**Review by Kurfursten — APPROVED** [review]
Code is fine

**Pull request merged**